engine/src/main/java/org/hibernate/validator/constraints/MinAge.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.constraints;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * The annotated element must be a date where the number of years go by to today must be
+ * greater or equal to the specified value
+ * <p>
+ * <p>
+ * The supported type is {@code LocalDate}. {@code null} is considered valid.
+ * <p>
+ *
+ * @author Hillmer Chona
```

Comment:
Isn't it Hilmer with one L? (same in the other classes)

That's how we wrote it in the copyright.txt file so we need to fix it if it's with 2 L.

Reply:
My name is Hillmer with double L

Reply:
I fixed it in copyright.txt, thanks. BTW, your email configuration says "Hilmer" when you're sending emails, that's why I got confused.